upgrade.go

```diff
@@ -305,15 +305,15 @@ func upgradePkgs(aurUp, repoUp upSlice) (stringSet, stringSet, error) {
 	ignore := make(stringSet)
 	aurNames := make(stringSet)
 
-	if len(aurUp)+len(repoUp) == 0 {
+	allUp := append(repoUp, aurUp...)
+
+	if len(allUp) == 0 {
 		return ignore, aurNames, nil
 	}
 
-	sort.Sort(repoUp)
-	sort.Sort(aurUp)
-	fmt.Printf("%s"+bold(" %d ")+"%s\n", bold(cyan("::")), len(aurUp)+len(repoUp), bold("Packages to upgrade."))
-	repoUp.Print(len(aurUp) + 1)
-	aurUp.Print(1)
+	sort.Sort(allUp)
```

Comment:
I'm not sure that `aur` will be the last in the every case there. Should I sort separately and append after this?

Reply:
Yeah sort separately then append

Reply:
Done.